app/models/concerns/foreman_build_history/host_extensions.rb

```diff
@@ -7,6 +7,7 @@ module HostExtensions
       after_build :set_build_start
       before_provision :set_build_finish
       before_save :build_history_handle_host_creation
+      has_one :build_history_facet, :foreign_key => 'host_id'
```

Comment:
I really like the idea of the plugin. We should make it official at some point.
It broke without the change in my local dev environment, though. Is there something else wrong?

```
NameError (undefined local variable or method `build_history_facet' for #<Host::Managed:0x007fedcdb3eda0>
```